### PR TITLE
tools.shtml

### PR DESCRIPTION
**Pull request by thijskh**

### CODE REVIEW

File: website/tools.shtml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+<p>
+    This page lists tools and facilities that can work with DOMjudge. They are
+    not in all cases maintained by the DOMjudge team.
+</p>
+
+
+<h2>Creating problemsets</h2>
+
+<h3>Checktestdata</h3>
```
Comment (mvr320) [inline website/tools.shtml:11]: Do you want to also link to:
https://github.com/RagnarGrootKoerkamp/BAPCtools
https://github.com/Kattis/problemtools

**Review by mvr320 — APPROVED** [review]

File: website/tools.shtml (hunk 1 above)
Comment (thijskh) [reply website/tools.shtml:11]: We accept packages created by either toolset, right? That seems like a good reason to list them.

File: website/tools.shtml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+<p>
+   <a href="checktestdata">checktestdata</a> is a program to syntactically verify
+   testdata according to a grammar file. It also supports generating testdata based
+   on a specification.
+</p>
+
```
Comment (mvr320) [inline website/tools.shtml:18]:
```suggestion
<h3>BAPCtools</h3>
<p><a href="https://github.com/RagnarGrootKoerkamp/BAPCtools">BAPCtools</a>  is a tool for creating and developing problems following the CLICS (DOMjudge/Kattis) problem format.<br>The aim of this tool is to run all necessary compilation, validation, and testing commands while working on an ICPC-style problem. Ideally one should never have to manually run any compilation or testing command themselves.</p>
```

File: website/tools.shtml (hunk 1 above)
Comment (mvr320) [reply website/tools.shtml:11]: Yes, I'll provide a suggestion for both.

File: website/tools.shtml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
```
Comment (mvr320) [inline website/tools.shtml:19]:
```suggestion
<h3>Problemtools</h3>
<p>The <a href="https://github.com/Kattis/problemtools">Kattis Problem Tool</a> is a collection of tools to create Kattis style problems. DOMjudge is able to import this format.</p>
```

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**